agent/acs/handler/heartbeat_handler.go

```diff
@@ -73,7 +79,17 @@ func (heartbeatHandler *heartbeatHandler) handleHeartbeatMessage() {
 }
 
 func (heartbeatHandler *heartbeatHandler) handleSingleHeartbeatMessage(message *ecsacs.HeartbeatMessage) error {
-	// Agent currently has no other action hooked to heartbeat messages, except simple ack
+	// Agent will run healthchecks triggered by ACS heartbeat
+	// healthcheck results will be sent on to TACS
+	dockerHealthcheck := doctor.NewDockerRuntimeHealthcheck(heartbeatHandler.taskEngine.TaskEngineClient())
+	heartbeatHandler.doctor.AddHealthcheck(&dockerHealthcheck)
+
+	go func() {
+		checkResult := heartbeatHandler.doctor.RunHealthchecks()
+		seelog.Infof("handler healthcheck result: %v", checkResult)
```

Comment:
Do we want to keep this log at info level or debug level as it will be a bit noisy?